numpy/random/mtrand/mtrand.pyx

```diff
@@ -3837,20 +3837,19 @@ cdef class RandomState:
         -------
         out : ndarray or scalar
             Drawn samples from the parameterized negative binomial distribution,
-            where each sample is equal to N, the number of trials it took to
-            achieve n - 1 successes, N - (n - 1) failures, and a success on the,
-            (N + n)th trial.
+            where each sample is equal to N, the number of failures that
+            occurred before a total of n successes was reached.
 
         Notes
         -----
         The probability density for the negative binomial distribution is
 
-        .. math:: P(N;n,p) = \\binom{N+n-1}{n-1}p^{n}(1-p)^{N},
+        .. math:: P(N;n,p) = \\binom{N+n-1}{N}p^{n}(1-p)^{N},
 
-        where :math:`n-1` is the number of successes, :math:`p` is the
-        probability of success, and :math:`N+n-1` is the number of trials.
-        The negative binomial distribution gives the probability of n-1
-        successes and N failures in N+n-1 trials, and success on the (N+n)th
+        where :math:`n` is the number of successes, :math:`p` is the
+        probability of success, and :math:`N+n` is the number of trials.
+        The negative binomial distribution gives the probability of n
+        successes and N failures in N+n trials, and a success on the (N+n)th
```

Comment:
Wikipedia uses "given" not "and" in this sentence - and the distinction is clearly important, because the table in [Alternative formulations](https://en.wikipedia.org/wiki/Negative_binomial_distribution#Alternative_formulations) shows different expressions for different orderings of "given"

I think this should say "the probability of N failures given n successes"

Reply:
Fixed!